lib/src/network/explorer/explorer_client.dart

```diff
@@ -286,4 +287,13 @@ class ExplorerClient {
       rethrow;
     }
   }
+
+  Future<PrioritiesEstimate> priority() async {
+    try {
+      return PrioritiesEstimate.fromJson(await _processGet(api('priority')));
+    } on ExplorerException catch (e) {
+      throw ExplorerException(
+          code: e.code, message: '{"priority": "{e.message}"}');
```

Comment:
For string interpolation we need to use `${element}`

`ExplorerException(code: e.code, message: '{"priority": "${e.message}"}');`